app/views/machine.njk

```diff
@@ -1,6 +1,20 @@
 {% extends 'layout.njk' %}
 
 {% block content %}
+
+<script>
+  let backupCalendar = [[], [], [], [], []];
```

Comment:
I would recommend moving this `<script>` block to the end of the file immediately before all the other `<script>` tags.